_includes/cockroachcloud/userfile-examples/backup-userfile.md

```diff
@@ -0,0 +1,37 @@
+We recommend starting backups from a time at least 10 seconds in the past using [`AS OF SYSTEM TIME`](../{{site.versions["stable"]}}/as-of-system-time.html). Read our guidance in the [Performance](../{{site.versions["stable"]}}/backup.html#performance) section on the [`BACKUP`](../{{site.versions["stable"]}}/backup.html) page.
+
+{{site.data.alerts.callout_info}}
+Only database and table-level backups are possible when using `userfile` as storage. Cluster-level backups will include `userfile` data as well, since this is also stored as table data in `defaultdb`.
```

Comment:
Regarding this note -- 

```
... Cluster-level backups will include userfile data as well, since this is also stored as table data in defaultdb
```.

In the second sentence, could we make it more clear that - Restoring cluster-level backups from a userfile will not work because the userfile is stores as a table in the defaultdb and you cannot restore a cluster with existing table data.

Reply:
Yep, definitely think this could be more defined. Adjusting.